dist/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LeaderBoard card</title>
+<script defer src="main.js"></script></head>
+
+<body>
+
+
+    <main>
+        <div class="list-add">
+
+            <h2> Add your score</h2>
+            <form action="#" method="get" id="list-form" class="add-form">
+                <input type="text" name="add-list" id="add-list" placeholder="add your name" required>
+                <input type="text" name="add-list" id="add-score" placeholder=" add your score" required>
+                <button type="submit" class="add-list-btn"> Submit <i class="fa-solid fa-check"></i></button>
+            </form>
+        </div>
+        <section class="to-do-list">
+            <h1> LeaderBoard </h1>
+            <div class="list-container">
+                <div class="list-heading">
+                    <h1 class="">Recent Scores </h1>
+                    <button type="button" class="refresh-list"><i class="fa-solid fa-arrows-rotate"></i></button>
+                </div>
+                <hr>
```

Comment:
- [ ] It is a requirement to follow the provided design, Kindly do so this includes the positioning of all the elements on the page.

Provided Design
![image](https://user-images.githubusercontent.com/81932478/174191830-ff98b3d7-f4b6-4b3e-8e99-318c217b2682.png)

Your Design
![image](https://user-images.githubusercontent.com/81932478/174191901-a4d007d1-123c-4ef4-a950-4e9035e4c88e.png)

- You could position the main header like it is in the design
- The recent scores and add scores headings should also be in the same line.